frontend/src/components/fields/WavenumberRangeSlider.tsx

```diff
@@ -77,7 +79,7 @@ export const WavenumberRangeSlider: React.FC<WavelengthRangeSliderProps> = ({
   return (
     <div>
       <Typography id="input-slider" gutterBottom>
-        Wavenumber range (cm⁻¹)
+        {isUnitChanged ? " Wavelength range(nm)" : " Wavenumber range(cm-1)"}
```

Comment:
```suggestion
        {isUnitChanged ? " Wavelength range (nm)" : " Wavenumber range (cm-1)"}
```
to make it look a little nicer